chef-encrypted-attributes.gemspec

```diff
@@ -32,7 +32,6 @@ Gem::Specification.new do |s|
   s.test_files = Dir.glob('{test,spec,features}/*')
   s.required_ruby_version = Gem::Requirement.new('>= 1.9.2')
 
-  s.add_dependency 'ffi-yajl', '>= 1.0', '< 3.0' # As Chef gem
```

Comment:
This needs to be updated in the README also. Something like the following:

* yajl-ruby `~> 1.1` or ffi_yajl `~> 1.0` (included with Chef)


https://github.com/onddo/chef-encrypted-attributes/blob/master/README.md#requirements